jobs/sync-stream-metadata.Jenkinsfile

```diff
@@ -14,25 +20,27 @@ cosaPod(configMaps: ["fedora-messaging-cfg"], secrets: ["fedora-messaging-coreos
         // XXX: eventually we want this as part of the pod or built into the image we use
         shwrap("git clone --depth=1 https://github.com/coreos/fedora-coreos-releng-automation /var/tmp/fcos-releng")
 
-        def cmd = """
-            aws s3 sync --acl public-read --cache-control 'max-age=60' \
-                --exclude '*' --include 'streams/*' --include 'updates/*' \
-                    ./ s3://fcos-builds
-        """
+        streams.production.each{
+            def cmd = """
+                aws s3 sync --acl public-read --cache-control 'max-age=60' \
+                    --exclude '*' --include 'streams/${it}.json' --include 'updates/${it}.json' \
+                        ./ s3://fcos-builds
+            """
 
-        // trim so that when we append --dryrun below, it's on the same line
-        cmd = cmd.trim()
+            // trim so that when we append --dryrun below, it's on the same line
+            cmd = cmd.trim()
 
-        // Do a dry run first to see if there's any work that actually needs to
-        // be done. `aws s3 sync` doesn't print anything if everything is
-        // synced up.
-        def dry_run = shwrapCapture("${cmd} --dryrun").trim()
-        if (dry_run != "") {
-            shwrap("${cmd}")
-            utils.shwrap("""
-            /var/tmp/fcos-releng/scripts/broadcast-fedmsg.py --fedmsg-conf=\${FEDORA_MESSAGING_CFG}/fedmsg.toml \
-                stream.metadata.update --stream ${params.STREAM}
-            """)
+            // Do a dry run first to see if there's any work that actually needs to
+            // be done. `aws s3 sync` doesn't print anything if everything is
+            // synced up.
+            def dry_run = shwrapCapture("${cmd} --dryrun").trim()
+            if (dry_run != "") {
+                shwrap("${cmd}")
+                utils.shwrap("""
+                /var/tmp/fcos-releng/scripts/broadcast-fedmsg.py --fedmsg-conf=\${FEDORA_MESSAGING_CFG}/fedmsg.toml \
+                    stream.metadata.update --stream ${it}
```

Comment:
dumb question.. Where is `${it}` defined?

Reply:
Is it an automatic variable that gets created from `streams.production.each{}` ?

Reply:
> Is it an automatic variable that gets created from `streams.production.each{}` ?

Yup, exactly! See https://riptutorial.com/groovy/example/32092/each-and-eachwithindex#example. I'll name it `stream` to be more explicit.